specs/date-picker/date-picker.md

```diff
@@ -0,0 +1,214 @@
+# Date Picker
+
+## Process
+
+- [ ] **Checkpoint 1** - Schedule a spec review meeting to cover the Overview section before proceeding to work on the Design section. Be sure to include relevant stakeholders such as members of the tools team.
+
+- [ ] **Checkpoint 2** - Schedule a spec review meeting before beginning the implementation work. Be sure to invite relevant stakeholders such as design system owners.
+
+- [ ] **Checkpoint 3** - Submit the component PR and schedule a component code review meeting or present the component in one of our Engineering Open Mic sessions. Address feedback from the review, obtain final approvals through GitHub, and merge.
+
+## Overview
+
+As defined by the W3C:
+>The date state represents a control for setting the element's value to a string representing a specific date.
+>
+
+### Background
+
+This is a community requested component.
+
+### Use Cases
+
+- John is booking a room online and needs to enter a date of arrival. Clicking on the calendar icon in the date field, brings up a calendar where John can select a check-in date.
+
+### Non-goals
+
+*A list of use cases, features, or functionality which are **not** goals for the component.*
+  
+### Features
+
+- **Date validation:** Verifies that the entered string is a valid date.
+
+- **Calendar view:** Opens a calendar view for selecting a date.
+
+- **Localization:** Formats that date for the local market. MM--DD-YYYY, DD--MM-YYYY, etc.
+
+### Risks and Challenges
+
+*Notable risks or challenges associated with implementing the component. Would we need to make any breaking changes in order to achieve this component's goals?*
+
+
+
+### Prior Art/Examples
+- [W3C](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/datepicker-dialog.html)
+- [FluentUI](https://developer.microsoft.com/en-us/fluentui#/controls/web/datepicker)
+
+---
+
+## Design
+
+*Describe the design of the component, thinking through several perspectives:*
+
+- *A customer using the component on a web page.*
+- *A developer building an app with the component and interacting through HTML/CSS/JavaScript.*
+- *A designer customizing the component.*
+
+### API
+*Component Name*
+- `fast-date-picker`
+
+*Props/Attrs*
+- `value` - A predefined date or one picked by the user.
```

Comment:
Is this component appropriate for a range of dates from an accessibility perspective? That's a common need, but not sure if we plan to address this here either initially or in the future.